calblueprint/siren
# Greg/multilingual

# ✨ New in this PR ✨

## One liner: what did you do? 🚨 
I set up the multilingual json dictionaries and created a language context provider using React Context, allowing user language preference. Right now, I only replaced the "Welcome" text on the home screen with a function that uses the language dictionaries (screens/HomeScreen.tsx, line 41).

## Coverage 🙆‍♀️
 - [x] Create the language json dictionaries
 - [x] Create the language context provider
 - [ ] Add in translations for each dictionary
 - [x] Replace all hard text with calls to language dictionaries
 - [x] Implement language selector 

## How can the reviewer test your code? 👩‍💻
Run `npm start` and get to the Home Screen to see the welcome message. Suraj and I still need to implement the language selector so that we can test using other languages. To test other areas of the app, I will probably just add in dummy translations, will do that for next PR.

## Any bugs you encountered or still having trouble with? 🐛
No

## Resources 📔
- https://reactjs.org/docs/context.html (React's documentation for Context)
- https://dev.to/halilcanozcelik/create-a-multi-language-website-with-react-context-api-4i27 (Tutorial on using Context for multilingual support)
- https://www.youtube.com/watch?v=kq9LqHKgNOA&t=42s (Video Tutorial on using Context for multilingual support)


🧜‍♀️ cc: @shannonbonet @phoebeli23
